cmd/rook/ceph/osd.go

```diff
@@ -326,8 +326,11 @@ func removeOSDs(cmd *cobra.Command, args []string) error {
 		return errors.Wrapf(err, "failed to parse --preserve-pvc flag")
 	}
 
+	forceRemovalCallback := func(x int) bool {
+		return forceOSDRemovalBool
```

Comment:
should we not have any check for OSDid, I see it exactly works how it was before

Reply:
yes, the intention is to works it exactly same as it was for upstream, the callback will be used by downstream tool